targets/PythonSdk/templates/API.py.ejs

```diff
@@ -0,0 +1,34 @@
+import json
+import PlayFabErrors
+import PlayFabHTTP
+import PlayFabSettings
+
+<%- generateApiSummary("", api, "description")%>
+<% for(var i in api.calls) { var apiCall = api.calls[i];
+%><%- generateApiSummary("", apiCall, "summary")
+%><%- getDeprecationAttribute("", apiCall)
+%>def <%- apiCall.name %>(request, callback, customData = None, extraHeaders = None):
+<%- getRequestActions("    ", apiCall) %>
+    def wrappedCallback(playFabResult, error):
+<%- getResultActions("        ", apiCall, api) %>
+        callback(playFabResult, error)
+
+    PlayFabHTTP.DoPost("<%- apiCall.url %>", request, <%- getAuthParams(apiCall) %>, extraHeaders, wrappedCallback)
+
+<% } %><% if (hasClientOptions) {
+%># Determine if the client session ticket is set, without actually making it public
+def IsClientLoggedIn():
+    return bool(PlayFabSettings._internalSettings.ClientSessionTicket)
+
+def MultiStepClientLogin(settingsForUser):
+    disabledAds = PlayFabSettings.DisableAdvertising
+    adIdType = PlayFabSettings.AdvertisingIdType
+    adIdVal = PlayFabSettings.AdvertisingIdValue
+
+    if settingsForUser["NeedsAttribution"] and not disabledAds and adIdType and adIdVal:
+        request = {}
+        if PlayFabSettings.AdvertisingIdType == AD_TYPE_IDFA:
```

Comment:
PlayFabSettings.AdvertisingIdType -> adIdType
(no use in making the temp variable otherwise)